### DISCUSSION TITLE
Event for creation/deletion of personal homes

### DISCUSSION THREAD

**Discussion by Camotoy** [EssentialsX/Essentials#5213] [Ideas and Feature Suggestions] [answered]
Same concept as https://github.com/EssentialsX/Essentials/issues/3868 - I'm looking for an event I can hook into that tells me when a player has created or deleted a home of theirs, without trying to parse the raw command.

**Comment by JRoy** [comment, marked as answer]
Will be added in https://github.com/EssentialsX/Essentials/pull/5216
